core-android/src/main/java/com/uber/sdk/android/core/auth/SsoDeeplink.java

```diff
@@ -138,7 +151,7 @@ private Uri createSsoUri() {
     @Override
     public boolean isSupported() {
         return appProtocol.isInstalled(activity, UBER, MIN_UBER_RIDES_VERSION_SUPPORTED)
-            || appProtocol.isInstalled(activity, UBER_EATS, MIN_UBER_EATS_VERSION_SUPPORTED);
+                || appProtocol.isInstalled(activity, UBER_EATS, MIN_UBER_EATS_VERSION_SUPPORTED);
```

Comment:
I think this has to fork based on productFlowPriority. Otherwise, you could get false positives. For example if the user has rides installed, but their productFlowPriority contains eats but not rides, then the deeplink would say SSO is supported even though it is not.